release-notes/opensearch-security.release-notes-2.4.0.0.md

```diff
@@ -0,0 +1,15 @@
+## 2022-11-04 Version 2.4.0.0
```

Comment:
Can you update this to 2022-11-10, the day of the release?

Reply:
(Note; I wouldn't be worried about the date alignment after this... we can always update the date post release)